.opencode/commands/create-skill.md

```
@@ -0,0 +1,8 @@
+---
+description: Scaffold a new agent skill following repository conventions.
+agent: build
+---
+
+Load the `create-skill` skill and follow its workflow to scaffold the new skill for: $ARGUMENTS
```

markdownlint-cli2 (0.21.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)